x-pack/plugins/observability/public/pages/home/index.tsx

```diff
@@ -0,0 +1,259 @@
+          }),
+        },
+        {
+          text: i18n.translate('xpack.observability.home.breadcrumb.gettingStarted', {
+            defaultMessage: 'Getting started',
+          }),
+        },
+      ]);
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    []
+  );
+
+  return (
+    <Container>
+      <Title>
+        <Page>
+          <EuiSpacer size="xxl" />
+          <EuiFlexGroup>
+            <EuiFlexItem grow={false}>
+              <EuiIcon type="logoObservability" size="xxl" />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiTitle size="m">
+                <h1>
+                  {i18n.translate('xpack.observability.home.title', {
+                    defaultMessage: 'Observability',
+                  })}
+                </h1>
+              </EuiTitle>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+          <EuiSpacer size="xxl" />
+        </Page>
+      </Title>
+      <Page>
+        <EuiSpacer size="xxl" />
+        <EuiFlexGroup direction="column">
+          {/* title and description */}
+          <EuiFlexItem style={{ maxWidth: 567 }}>
+            <EuiTitle size="s">
+              <h2>
+                {i18n.translate('xpack.observability.home.sectionTitle', {
+                  defaultMessage: 'Observability built on the Elastic Stack',
+                })}
+              </h2>
+            </EuiTitle>
+            <EuiSpacer size="m" />
+            <EuiText size="s" color="subdued">
+              {i18n.translate('xpack.observability.home.sectionsubtitle', {
+                defaultMessage:
+                  'Bring your logs, metrics, and APM traces together at scale in a single stack so you can monitor and react to events happening anywhere in your environment.',
+              })}
+            </EuiText>
+          </EuiFlexItem>
+
+          {/* Apps sections */}
+          <EuiFlexItem>
+            <EuiSpacer size="s" />
+            <EuiFlexGroup>
+              <EuiFlexItem>
+                <EuiFlexGrid columns={2}>
+                  {appsSection.map((app) => (
+                    <Section section={app} key={app.id} />
+                  ))}
+                </EuiFlexGrid>
+              </EuiFlexItem>
+              <EuiFlexItem>
+                <EuiImage
+                  size="xl"
+                  alt="observability overview image"
+                  url={core.http.basePath.prepend(
+                    '/plugins/observability/assets/observability-overview.png'
+                  )}
+                />
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+
+          {/* Get started button */}
+          <EuiFlexItem>
+            <EuiFlexGroup justifyContent="center" gutterSize="none">
+              <EuiFlexItem grow={false}>
+                <EuiButton
+                  fill
+                  iconType="sortRight"
+                  iconSide="right"
+                  style={{ width: 175 }}
```

Comment:
Is this width missing units? And why 175? If the user changes locale the "Get started" string will be of different length (could be much longer)

Reply:
It's not needed, removed.